components/cd-form/cd-form--drupal-widget.css

```diff
@@ -0,0 +1,204 @@
+  margin-top: 1rem;
+}
+
+/* Image and/or file field */
+form .form-managed-file {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
+
+@media (min-width: 576px) {
+  form .form-managed-file {
+    flex-direction: row;
+    align-items: center;
+  }
+}
+/* Media image field */
+form .image-widget article {
+  display: flex;
+  flex: 1 0 auto;
+  flex-direction: column;
+  order: 1;
+}
+/* File name */
+@media (min-width: 576px) {
+  form .image-widget span.file,
+  form .image-widget span.file-size {
+    margin-inline-start: 0.5rem;
+  }
+  form .image-widget span.file-size {
+    margin-inline-end: 0.5rem;
+  }
+}
+
+form .image-widget article > div {
+  margin-bottom: 1rem;
+}
+
+@media (min-width: 576px) {
+  form .image-widget span.file,
+  form .image-widget article > div {
+    margin-bottom: 0;
+  }
+}
+form .image-widget .js-media-library-item-preview {
+  margin: 0 0 1rem;
+}
+
+form .image-widget .form-submit,
+form .form-managed-file .form-submit {
+  order: 2;
+}
+
+@media (max-width: 575px) {
+  form .image-widget .form-submit,
+  form .form-managed-file .form-submit {
+    margin-top: 1rem;
+  }
+}
+
+.form-managed-file img,
+.form-managed-file .file {
+  margin-inline-end: 0.5rem;
+}
+/* Alt text field and/or file description field */
+.form-managed-file .form-type-textfield {
+  flex: 1 0 100%;
+  order: 4;
+  margin-top: 0.5rem;
+}
+.form-managed-file .form-type-checkbox {
+  padding-bottom: 0;
+}
+/* Image title field */
+.form-managed-file .form-type-textfield + .form-type-textfield {
+  order: 5;
+}
+
+/* Show/Hide media item weight button */
+.js-media-library-widget-toggle-weight {
+  position: absolute;
+  right: 1rem;
+  padding: 0.25rem;
+  font-size: var(--cd-font-size--small);
+}
+/* Media item weight */
+form .image-widget .form-type-number {
+  flex: 0 1 auto;
+  order: 3;
+}
+form .image-widget .form-type-number input {
+  width: 100%;
+}
+
+@media (min-width: 576px) {
+  form .image-widget .form-type-number {
+    flex: 0 1 100px;
+    margin-top: 0;
+    padding: 0 0 0 1rem;
+  }
+}
+/* Mutliple files and/or images displayed in a table */
+.tabledrag-toggle-weight-wrapper {
+  position: relative;
+  margin-bottom: 1rem;
+}
+/* Show/Hide media item weight button */
+.tabledrag-toggle-weight {
+  padding: 0.25rem;
+  font-size: var(--cd-font-size--small);
+}
+/* Table drag */
+a.tabledrag-handle .handle {
+  width: auto;
+  height: auto;
+}
+
+/* Media library dialog */
+.media-library-widget-modal {
+  /* Higher than CD header elements */
+  z-index: 202;
+}
+
+ul.js-media-library-add-form-added-media {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+ul.js-media-library-add-form-added-media li {
+  margin: 0;
+  padding: 0;
+}
+.js-media-library-add-form-added-media .form-submit {
+  margin-top: 1rem;
+}
+.ui-dialog .ui-dialog-buttonpane button {
+  margin: 0;
+  border-width: 2px;
+  border-color: inherit;
+}
+.js-media-library-item {
```

Comment:
```suggestion
.js-media-library-item.views-row {
```

The selector is too broad and the flex rules break the media library when displayed as a table instead of a grid (see RW9, blog post for example):

<img width="1116" alt="Screen Shot 2021-12-16 at 12 40 16" src="https://user-images.githubusercontent.com/696348/146304217-e00bc927-4b22-4d01-958d-a5d089d82723.png">

Reply:
Thanks, I've added a commit that includes another selector as well. The `.js-media-library-item` is also the class used when the media fields are displayed on the forms.

Reply:
Thanks!